src/components/Inventory/view/BatchesExportModal.tsx

```diff
@@ -0,0 +1,26 @@
+import { SearchNodePayload, SearchSortOrder } from 'src/types/Search';
+import { NurseryBatchService } from 'src/services';
+import ExportCsvModal from 'src/components/common/ExportCsvModal';
+
+interface DownloadReportModalProps {
+  organizationId: number;
+  speciesId: number;
+  searchFields: SearchNodePayload[];
+  searchSortOrder?: SearchSortOrder;
+  onClose: () => void;
+}
+
+export default function DownloadReportModal(props: DownloadReportModalProps): JSX.Element {
```

Comment:
Should we call the component BatchesExportModal? And BatchesExportModalProps ?

Reply:
Yes, we should - thanks :)
Will fix it.